### PR TITLE
Issue #8: Hide social media when footer is toggle

### PR DESCRIPTION
**Pull request by danielcb29**
Hey guys!

This PR solves the Issue #8 , it includes: 

- Footer output to dispatch event the footer state change 
- Home function to handle the media show
- Animation to hide the media (check this item please) 

Any change just let me know! ✋ 

Thanks! 🔥 🚀

### CODE REVIEW

**Comment by danielcb29** [comment]
Let me update the branch

File: src/app/home/home.component.ts [hunk 1]
```diff
@@ -1,13 +1,22 @@
...
+    ]),
+  ]
 })
 export class HomeComponent {
 
-  
+  showMedia = true;
+
+  onFooterToogle(infoState: string) {
+    this.showMedia = infoState === 'hide' ? true : false;
```
Comment (jdjuan) [inline src/app/home/home.component.ts]: `this.showMedia = infoState === 'hide';` This is enough 😉

Comment (danielcb29) [reply src/app/home/home.component.ts]: It's true!

**Review by jdjuan — APPROVED** [review]

**Comment by danielcb29** [comment]
Branch updated @jdjuan 🚀

**Pull request merged**